soh/soh/Enhancements/debugger/actorViewer.cpp

```diff
@@ -26,6 +26,7 @@ extern PlayState* gPlayState;
 
 #include "textures/icon_item_static/icon_item_static.h"
 #include "textures/icon_item_24_static/icon_item_24_static.h"
+#include <soh/ActorExtension/ActorListIndex.h>
```

Comment:
When importing our own files from within the src, use quotes instead of brackets
```suggestion
#include "soh/ActorExtension/ActorListIndex.h"
```